clients/go/events/errors/errors.go

```diff
@@ -31,6 +32,18 @@ func (r EventError) Error() string {
 	return fmt.Sprintf("%s: %s, caused by [%s]", r.Code, r.Message, r.Cause.Error())
 }
 
+func (r *EventError) Is(target error) bool {
+	t, ok := target.(*EventError)
+	if !ok {
+		return false
+	}
```

Comment:
The one additional check I would add is:
```suggestion
	}
	
	if r == nil && t == nil {
	    return true
	}
	
	if r == nil || t == nil {
	    return false
	}
```

Example for why: https://play.golang.org/p/P85Pext7xw5

Reply:
Very interesting, it doesn't seem like the check should be necessary, but the example certainly clarifies it.